scripts/run.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+
+# Initialize variables
+script=""
+autoRunLoc=$(readlink -f "$0")
+proc_name="auto_run_validator" 
+args=()
+version_location="./openvalidators/__init__.py"
+version="__version__"
+
+args=$@
+
+# Check if pm2 is installed
+if ! command -v pm2 &> /dev/null
+then
+    echo "pm2 could not be found. To install see: https://pm2.keymetrics.io/docs/usage/quick-start/"
+    exit 1
+fi
+
+# Checks if $1 is smaller than $2
+# If $1 is smaller than or equal to $2, then true. 
+# else false.
+versionLessThanOrEqual() {
+    [  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]
+}
+
+# Checks if $1 is smaller than $2
+# If $1 is smaller than $2, then true. 
+# else false.
+versionLessThan() {
+    [ "$1" = "$2" ] && return 1 || versionLessThanOrEqual $1 $2
+}
+
+# Returns the difference between 
+# two versions as a numerical value.
+get_version_difference() {
```

Comment:
may we keep`naming consistency?
i think bash scripts are expected to have snake cases naming, what do you think about that?

Reply:
Done!